stl/src/filesystem.cpp

```diff
@@ -434,18 +434,35 @@ void __stdcall __std_fs_directory_iterator_close(_In_ const __std_fs_dir_handle
     return __vcp_Copyfile(_Source, _Target, /* _Fail_if_exists = */ false);
 }
 
-_Success_(return == __std_win_error::_Success) __std_win_error
-    __stdcall __std_fs_get_file_id(_Out_ __std_fs_file_id* const _Id, _In_z_ const wchar_t* const _Path) noexcept {

```

Comment:
I don't think `__std_fs_get_file_id` can be removed from import library; it has to be preserved due to ABI.
See `TRANSITION, ABI: preserved for binary compatibility` comment in many places here

Reply:
Yeah, I think you are right. I restored the function in the cpp file and also moved the definition of the type `__std_fs_file_id` there.